test/GitHub.InlineReviews.UnitTests/Models/DiffUtilitiesTests.cs

```diff
@@ -366,23 +366,6 @@ public void CountCarriageReturns_NullText_ArgumentNullException()
             {
                 Assert.Throws<ArgumentNullException>(() => DiffUtilities.LineReader.CountCarriageReturns(null));
             }
-
-            static void Benchmark_ReadLineAndCountCarriageReturns_100000()
-            {
-
-                var file = new System.Diagnostics.StackFrame(true).GetFileName();
```

Comment:
This was an ad-hoc test method I ran using TestDriven.Net. Easy for quick benchmarking. Zapped it now.